src/main/scala/com/mozilla/telemetry/stats/MannWhitneyU.scala

```diff
@@ -0,0 +1,120 @@
+/*
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.mozilla.telemetry.stats
+
+
+// Constants
+
+object MannWhitneyU {
+
+  def erf(x: Double): Double =  {
+    // Constants.
+    val a1: Double = 0.254829592
```

Comment:
Please remove types for values and variables. [See the types style guide](https://docs.scala-lang.org/style/types.html).